shared/params.go

```diff
@@ -140,39 +140,22 @@ const MaxCountMaxValue = 500
 // MaxCountMinValue is the minimum allowed value for the max-count param.
 const MaxCountMinValue = 1
 
-// SHARegex is a regex for SHA[0:10] slice of a git hash.
-var SHARegex = regexp.MustCompile("[0-9a-fA-F]{10,40}")
+// SHARegex is a regex for 7 to 40 char prefix of a git hash.
```

Comment:
Why is the minimum 7? Is it trying to make sure SHA is unique? What'll happen is the prefix isn't unique?

Reply:
Don't have a great reason, but the shorter we get, the more likely to clash. If it's not unique, it'll just match more runs. Do you prefer a different minimum?